stable/locust/Chart.yaml

```diff
@@ -1,5 +1,11 @@
+apiVersion: v1
 name: locust
-description: A chart for locust distributed load testing
-version: 0.1.0
+description: A modern load testing framework
+version: 0.1.1
+maintainers:
+  - name: Vincent De Smet
```

Comment:
I actually have a similar PR open already - https://github.com/kubernetes/charts/pull/881 - should I update the description / version and add my name there?